files/register_route53.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+until ping -c 1 google.com &> /dev/null; do
+  echo "waiting for outbound connectivity"
+  sleep 5
+done
+
+# Register the instance in Route 53, using the volume id for the sub-domain
+
+subdomain="$( echo -n "$volume_id" | sed 's/^vol-//' )"
```

Comment:
I think it would be best if this script queries the volume id instead of relying on the variable.

Reply:
I have set up a query that will ask for a value for volume_id instead of depending on a variable from the ebs_volume.sh script.